jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -158,7 +158,8 @@ public void close() {
     sampler.close();
   }
 
-  private class SpanBuilder implements io.opentracing.Tracer.SpanBuilder {
+  //Visible for testing
```

Comment:
Because it is no longer a unit test. `SpanBuilder#isRPCServer` is implemented in `SpanBuilder` and not in `Span`. 

`Span` has it's own separate implementation of when it's representing an rpc or client span. I feel that we should document and refactor these classes at some point.